src/engine/Operation.cpp

```diff
@@ -146,12 +142,12 @@ shared_ptr<const ResultTable> Operation::getResult(bool isRoot,
         // Note: both of the following calls have no effect and negligible
         // runtime if neither a LIMIT nor an OFFSET were specified.
         result.applyLimitOffset(_limit);
-        _runtimeInfo.addLimitOffsetRow(_limit, limitTimer.msecs(), true);
+        _runtimeInfo->addLimitOffsetRow(_limit, limitTimer.msecs(), true);
       } else {
         AD_CONTRACT_CHECK(result.idTable().numRows() ==
                           _limit.actualSize(result.idTable().numRows()));
       }
-      return CacheValue{std::move(result), getRuntimeInfo()};
+      return CacheValue{std::move(result)};
```

Comment:
The CacheValue wrapper is not really required anymore now that it's only wrapping a single shared pointer

Reply:
We'll have to discuss what it is anyway:)